flask_app/blueprints/api/replication.py

```diff
@@ -8,11 +8,11 @@
 NoneType = type(None)
 
 
-@API
+@API(version=2)
 @requires_role('admin')
-def create_replication(url: str, username: (str, NoneType)=None, password: (str, NoneType)=None):
+def create_replication(url: str, index_name: (str, NoneType)='backslash', username: (str, NoneType)=None, password: (str, NoneType)=None):
```

Comment:
This allows `index_name` to be `None`, but the database would forbid it....

Reply:
@vmalloc not sure i understand. this lets not passing  index_name parameter to create_replication, and in such case the index will be 'backslash'. Why is it wrong?

Reply:
if the API receives None as a parameter, it will be passed as NULL to the DB, at which point I don't think the default gets used...

Reply:
i see, thanks! pushed a fix.